.github/workflows/include-kcl-checksums.yml

```diff
@@ -0,0 +1,48 @@
+name: Include KCL Modules Checksum
+
+on:
+  workflow_dispatch:
+    inputs:
```

Comment:
There should be one more input, select a particular package, add the checksum of that package, and allow each run to apply to only one package rather than all of them.

Reply:
Could you specify the required input format? Should it include only the package name and version? Since I am relying on the package's location within a GitHub repository, would it be appropriate to recursively search through the repository to gather all the necessary information about the package?

Reply:
For now, it is enough to solve the problem of the current repository, that is, to locate the package using the package `name` and `version`.